test/integration_test/basic_test.go

```diff
@@ -307,6 +308,49 @@ func BasicInstall(tc *types.TestCase) func(*testing.T) {
 	}
 }
 
+func TestBasicInstall(t *testing.T) {
+	testCase := &types.TestCase{
+		TestName: "BasicInstall",
+		TestSpec: func(t *testing.T) interface{} {
+			objects, err := ci_utils.ParseSpecs("storagecluster/storagecluster-with-all-components.yaml")
+			require.NoError(t, err)
+			cluster, ok := objects[0].(*corev1.StorageCluster)
+			require.True(t, ok)
+			cluster.Name = "test-basic-stc"
+			cluster.Namespace = ci_utils.PxNamespace
+			return cluster
+		},
+		TestFunc: BasicInstallAetosDemo,
+	}
+	BasicInstallAetosDemo(testCase)(t)
+}
+
+func BasicInstallAetosDemo(tc *types.TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*corev1.StorageCluster)
+		dash.VerifyFatal(t, ok, true, fmt.Sprintf("Get StorageCluster from test spec"))
+
+		defer dash.TestCaseEnd()
```

Comment:
below code should be the first step . so that no logs will be missed to push to aetos
```
defer dash.TestCaseEnd()
dash.TestCaseBegin("BasicInstall", "Deploying and validating StorageCluster", "", nil)

```